javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,135 @@
+//1-Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+const callText = (sec) => {
+    setTimeout(() => { console.log(`Called after ${sec} seconds`) }, sec * 1000);
+}
+
+callText(2.5)
+
+//2- Create a function that takes 2 parameters: delay and stringToLog. 
+//Calling this function should log out the stringToLog after delay seconds. 
+//Call the function you have created with some different arguments. 
+
+const LogString = (delay, stringToLog) => {
+
+    setTimeout(() => { console.log(stringToLog) }, delay * 1000);
+}
+
+LogString(6, "This string logged after 6 seconds")
+LogString(10, "This string logged after 10 seconds")
+
+//3-Create a button in html. When clicking this button, use the function 
+//you created in the previous task to log out the text:Called after 5 seconds 5 seconds after the button is clicked.
+
+const main = document.createElement('main')
+document.body.appendChild(main)
+const delayButton = document.createElement('button');
+
+delayButton.setAttribute("id", "btn");
+delayButton.innerHTML = 'Click';
+main.appendChild(delayButton);
+const p1 = document.createElement('p');
+document.body.appendChild(p1)
+delayButton.addEventListener('click', () => {
+    callText(5);
+})
+
+//4- planet Logger Function
+const earthLogger = () => {
+    console.log("Earth");
+}
+const saturnLogger = () => {
+    console.log("Satrun");
+}
+
+const planetLogFunction = (fun) => {
+    fun === earthLogger ? earthLogger() : saturnLogger();
+}
+
+planetLogFunction(earthLogger)
+planetLogFunction(saturnLogger)
+
+// 5- Create a button with the text called "Log location".The location (latitude, longitude) of the user.
+const locationButton = document.createElement('button');
```

Comment:
this is a nice practice to create elements in js instead of using html good job Haile